03_simon_says/simon_says.rb

```diff
@@ -1 +1,60 @@
-#write your code here
+def echo(greeting)
+    greeting.downcase
+end
+
+def shout(word)
+    word.upcase
+end
+
+def repeat(double_word, number_of_times = nil)
+    repeats = ""
+    if number_of_times != nil && number_of_times != 0
+        i = 0
+        while i < number_of_times - 1 do
+            repeats += double_word.downcase + " "
+            i += 1
+        end
+        repeats += double_word.downcase
+    end
+    if number_of_times == nil
+        repeats = double_word.downcase + " " + double_word.downcase
+    end
+    repeats = repeats.to_s
```

Comment:
If `number_of_times` variable is empty, we can just return the `double_word` variable at the beginning of the function since we know that it does not repeat. For example:
`return double_word if numbers_of_times === nil`